lib/manageiq/messaging.rb

```diff
@@ -1,4 +1,5 @@
 require 'active_support/core_ext/module/delegation'
+require 'active_support/core_ext/hash'
```

Comment:
For `slice`.  I'm not concerned about adding active_support, since we have more_core_extensions anyway, which, in turn, requires active_support, and even without that activesupport doesn't bother me